index.css

```diff
@@ -65,14 +65,28 @@ h2 {
 }
 
 /**BUTTONS**/
-.reward-btn {
-    background-color: #9E4770;
+.reward-btns {
+    display: flex;
+    margin: 0;
+    justify-content: space-between;
+    width: inherit;
+    
+    div {
```

Comment:
This is being applied to every div, if we make a div in the future we don't want to have half width, that could be a problem, maybe make a class like width-50 or smthin?

Reply:
That's a good idea, I'm make the change!